config/routes.rb

```diff
@@ -9,6 +9,7 @@
     delete "/logout", to: "sessions#destroy"
     get "/search", to: "search#index"
     get "categories", to: "categories#index"
```

Comment:
cái này sử dụng resources được không, cái nào theo REST thì theo đúng nhé

Reply:
bọn em truyền slug nên k dùng resource anh ạ

Reply:
@duongtt-1061 ý anh là nếu sử dụng các default action (index,...) của controller mà không thay đổi đường dẫn như line 9,10 thì nên  dùng resource/ resources

có thể tìm hiểu thêm về resource/resources và route in [doc](https://guides.rubyonrails.org/routing.html)

Reply:
vâng anh